clients/java-logger/library/src/main/java/com/abcxyz/lumberjack/auditlogclient/processor/JustificationProcessor.java

```diff
@@ -121,6 +127,26 @@ public AuditLog.Builder auditLogBuilderWithJustification(
           Value.newBuilder().setStructValue(justificationStruct).build());
       auditLogBuilderCopy.setMetadata(metadataBuilder.build());
 
+      Claim justificationsClaim = jwt.getClaim("justs");
```

Comment:
Can we add a comment that the logic here is no justs and then continue with other the populate of other fields?
Context: https://github.com/abcxyz/lumberjack/pull/293#discussion_r971292134

Reply:
I have added a change to return early when justs is not found or null or empty, do you think it is enough?

Reply:
Ok, I just see your comment there, for the concern that missing populating other new fields, I think unit test will catch that, what do you think?

Reply:
Why the unit tests will catch it?For example, later we when want to populate another field called "C", the developer need to know the logic is "C" filed still need to get populated even though there is no justs found. The developer need to know the logic and then modify the UT to catch that. Our current UT won't be able to catch it.

Reply:
Right, current UT should only cover existing functionalities, if adding a new field, the developer should add UT to test that is what I meant.

Reply:
For the developers to add UT, they need to know the business logic is to continue when there is no justs found. That's why I asked to add a comment. Currently the code looks unclear whether the logic is to exit or to continue.

Reply:
Added to avoid future confusions, please check newest commit, thanks!

Reply:
Can we add a Java comments instead of adding it in the logging?

Reply:
Ack.